manifests/otel-agent-cm.yaml

```diff
@@ -26,43 +26,27 @@ data:
   otel-agent-config.yaml: |
     receivers:
       opencensus:
+        endpoint: 0.0.0.0:55678
     exporters:
       opencensus:
         endpoint: otel-collector.config-management-monitoring:55678
         tls:
           insecure: true
     processors:
-      # Attributes processor adds custom configsync metric labels to applicable
```

Comment:
Why did these attributes get removed from the agent config?

Reply:
It seems to be replaced by `manifests/otel-agent-reconciler-cm.yaml`